Vagrantfile

```diff
@@ -69,13 +70,14 @@ Vagrant.configure("2") do |config|
       'scripts'        => [ $ubuntu_deps, ],
       'fix_console'    => 0,
     },
-    'ubuntu-21.04'     => {
-      'image'          => 'ubuntu/hirsute64',
+    'ubuntu-22.04'     => {
```

Comment:
Oh actually I forgot `llvm.sh` doesn't work on 22.04. So might as well just delete this.